zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -559,21 +581,23 @@ def channel_message_to_zerver_message(realm_id: int, users: List[ZerverFieldsT],
                                       all_messages: List[ZerverFieldsT],
                                       zerver_realmemoji: List[ZerverFieldsT],
                                       zerver_subscription: List[ZerverFieldsT],
-                                      added_channels: AddedChannelsT,
+                                      added_channels: AddedChannelsT, id_list: List[int],
                                       domain_name: str) -> Tuple[List[ZerverFieldsT],
                                                                  List[ZerverFieldsT],
                                                                  List[ZerverFieldsT],
                                                                  List[ZerverFieldsT],
-                                                                 List[ZerverFieldsT]]:
+                                                                 List[ZerverFieldsT],
+                                                                 List[int]]:
```

Comment:
Are these changes independent of the other change?

Reply:
This is because `id_list` is returned from this function. It is not independent.